monitor/alerts/list-get-example-all/list-get-example-all.3.x.js

```diff
@@ -0,0 +1,9 @@
+// Download the Node helper library from twilio.com/docs/node/install
+// These are your accountSid and authToken from https://www.twilio.com/console
+const accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
+const authToken = 'your_auth_token';
+
+const client = require('twilio')(accountSid, authToken);
+
+client.monitor.v1.alerts.list()
```

Comment:
I would use #each method provided in the sdk: https://github.com/TwilioDevEd/api-snippets/blob/902701e1c7c9ca953969d47af6c20ca8451f952c/rest/message/list-get-example-1/list-get-example-1.3.x.js#L7